app/vmalert/notifier/notifier_blackhole.go

```diff
@@ -0,0 +1,49 @@
+package notifier
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/app/vmalert/utils"
+)
+
+type blackHoleMetrics struct {
+	alertsSent *utils.Counter
+}
+
+func newBlackHoleMetrics(addr string) *blackHoleMetrics {
```

Comment:
I suggest using the existing function `newMetrics()`, so the metric names will be declared at one place. Otherwise, there is a probability metric names or labels will diverge with time

Reply:
ok sure.